packages/app-plugin-security-cognito-theme/package.json

```diff
@@ -10,6 +10,7 @@
     "@webiny/app-plugin-security-cognito": "^4.12.1",
     "@webiny/app-security": "^4.12.1",
     "@webiny/form": "^4.12.1",
+    "@webiny/plugins": "^4.12.1",
```

Comment:
Not sure if it should also be added in both `tsconfig.json` and `tsconfig.build.json` :thinking: 
What do you think @Pavel910?

Reply:
@brunozoric @Ashu96 is right, every in-repo dependency must be added to tsconfig files as well, as references.

Reply:
added plugins to references in both `tsconfig.json` and `tsconfig.build.json`